Extensions/editable_reblogs.js

```diff
@@ -358,6 +358,13 @@ XKit.extensions.editable_reblogs = new Object({
 					XKit.interface.kitty.set(response.getResponseHeader("X-tumblr-kittens"));
 					XKit.tools.add_function(function() {
 						Tumblr.Events.trigger("postForms:saved");
+						var query_string = window.location.search;
+						if (query_string.indexOf("redirect_to=") !== -1) {
+							var redirect_url = window.location.search.split("redirect_to=")[1];
```

Comment:
I wasn't referring specifically to param, but just that I'm pretty sure
there's a jquery function to do it

On Sat, Dec 5, 2015 at 8:25 PM Bryan Tsang <notifications@github.com> wrote:

> In Extensions/editable_reblogs.js
> <https://github.com/new-xkit/XKit/pull/832#discussion_r46765394>:
>
> > @@ -358,6 +358,13 @@ XKit.extensions.editable_reblogs = new Object({
> >  					XKit.interface.kitty.set(response.getResponseHeader("X-tumblr-kittens"));
> >  					XKit.tools.add_function(function() {
> >  						Tumblr.Events.trigger("postForms:saved");
> > +						var query_string = window.location.search;
> > +						if (query_string.indexOf("redirect_to=") !== -1) {
> > +							var redirect_url = window.location.search.split("redirect_to=")[1];
>
> I get this doesn't handle the case where redirect_to isn't the only
> search parameter, but can you help me understand how $.param
> <https://api.jquery.com/jquery.param> helps? It takes an array or an
> object and returns its serialized representation, but we want is the
> reverse: to deserialize the query parameter and redirect to it.
>
> The solution I'm thinking of will probably be something like this
> <http://stackoverflow.com/a/901144/2073440>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/832/files#r46765394>.
>